src/test/unit/engine/HelloRegionTest.cpp

```diff
@@ -50,7 +55,7 @@ int main(int argc, const char *argv[]) {
 
   // Load data
   std::string path =
-      Path::makeAbsolute("../../../src/examples/regions/Data.csv");
+      Path::makeAbsolute("../release/data/Data.csv"); //FIXME use path relative to CMake's nupic root
```

Comment:
FIXME: how do I address this file somehow absolutely? If this is run by hand from ./build/scripts/, this works; if ran from make tests_all, it has apparently a different relative location. Do we have some variable that addresses the root of the repo?